backend/e4h-services/ingestion-service/app/utils/fieldplan_service_client.py

```
@@ -29,45 +30,66 @@ def create_fieldPlan_facility(self, request_info: RequestInfo, fieldPlan_id: str
                 'tenantId': 'in'
             }
         }
+        logger.trace(f"Creating field plan facility: fieldplan_id={fieldPlan_id}, facility_id={facility_id}")
         try:
             response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)